### DISCUSSION TITLE
Cannot properly attach Node debugger to vanilla "my-remix-app"

### DISCUSSION THREAD

**Discussion by kingstonfung** [remix-run/remix#9053] [Q&A]
Hello,

I am following the doc (https://remix.run/docs/en/main/future/vite#debugging) here about Debugging, using a vanilla `"my-remix-app"` and a Chrome debugger tool.. I noticed the debugging behaves differently when attaching to my development session.

- When launching with  `NODE_OPTIONS="--inspect-brk" npm run dev` , I was only able to connect with the debug session but it does not halt operations upon `debugger;` statements, nor it shows any of my source files in the tree.

- However, under the same development conditions but launched with `node --inspect-brk node_modules/.bin/remix vite:dev`... my debug session behaves fine as I was able to attach the debugger, having it respect to `debugger;` statements as well as any impromptu breakpoints selected on the source files (because I can see the files, and was able to click through them)

Curious if debugging with Chrome Dev Tools is still a work-in-progress and is a known issue, or if this is probably a fresh problem?

Launching my debug session via VSCode with `NODE_OPTIONS="--inspect-brk" npm run dev` from launch.json have no issues. VSCode worked perfectly fine without reaching into node_modules.

Thanks very much!

### Steps to reproduce
1. Following the [Getting Started Guide](https://remix.run/docs/en/main/future/vite#getting-started)
  i. Select the most simplistic setup, "minimal server" option as written
  i. `npx create-remix@latest --template remix-run/remix/templates/vite`
2. Start up your dev server with `NODE_OPTIONS="--inspect-brk" npm run dev`
3. Attaching Chrome Dev Tools to your dev session
4. Notice the debugger would attach, however, source files and `debugger;` statements would not be available or acknowledged 
5. Then, restart your debug session with `node --inspect-brk node_modules/.bin/remix vite:dev`
6. Attach your session with Chrome Dev Tools once again, this time, debugging works.

*Remix 2.8.1 is the latest version as of writing*

**Comment by albertdbio** [comment]
Is this not going to get answered?
